builder/virtualbox/common/step_export.go

```diff
@@ -22,11 +22,28 @@ type StepExport struct {
 	Format         string
 	OutputDir      string
 	ExportOpts     []string
+	Bundling       VBoxBundleConfig
 	SkipNatMapping bool
 	SkipExport     bool
 }
 
 func (s *StepExport) Run(_ context.Context, state multistep.StateBag) multistep.StepAction {
+	// If ISO export is configured, ensure this option is propagated to VBoxManage.
+	if s.Bundling.BundleISO {
+		foundISOOption := false
+
+		for _, option := range s.ExportOpts {
+			if option == "--iso" || option == "-I" {
+				foundISOOption = true
+				break
+			}
+		}
+
+		if !foundISOOption {
+			s.ExportOpts = append(s.ExportOpts, "--iso")
```

Comment:
instead of messing around with the foundISOOption variable, why not just move this line up into the original loop?  

```
for _, option := range s.ExportOpts {
		if option == "--iso" || option == "-I" {
			s.ExportOpts = append(s.ExportOpts, "--iso")
			break
		}
}
```

Reply:
Feel free to submit a PR for that change lol

Reply:
That’s a good improvement, will add this when I get around to resolving the merge conflict